frontend/src/pages/Admin/Components/Resource/Resource.jsx

```diff
@@ -1,9 +1,69 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { END_POINT } from "../../../../config/api";
+import axios from "axios";
+import "./resource.module.scss";
+import style from "./resource.module.scss";
 
 export function Resource() {
+  const [resourses, setResources] = useState([]);
```

Comment:
Please add the three-dotted loader during the api call.

Reply:
ok on it